http_prompt/contextio.py

```diff
@@ -14,21 +12,14 @@
 # Don't save these HTTPie options to avoid collision with user config file
 EXCLUDED_OPTIONS = ['--style']
 
-
-def url_to_context_filename(url):
-    r = urlparse(url)
-    host = r.hostname
-    port = r.port
-    if not port:
-        port = 443 if r.scheme == 'https' else 80
-    return host + '.' + str(port)
+# Filename the current environment context will be saved to
+CONTEXT_FILENAME = 'context.hp'
```

Comment:
Currently, we store context data in json. For consistency, we may want to change it to the same format with `env > file.txt` (let's call this "http-prompt script"). But it can be done in another PR. For now, let's keep it in json in this PR.